istio/assets/dashboards/istio_1_5_overview

```diff
@@ -16,11 +16,17 @@
         "verified": true
     },
     "description": "### Istio\n\nNote: This dashboard outlines new metrics from Istio `v1.5+.\n\n- [What is Istio](https://istio.io/docs/concepts/what-is-istio/)?\n- [Istiod](https://istio.io/docs/ops/deployment/architecture/)\n- [Envoy](https://istio.io/docs/concepts/what-is-istio/#envoy), a high performance proxy.\n    - XDS = Envoy management APIs that can be implemented by backend servers. \n        - These APIs support:\n            - Cluster discovery service (CDS)\n            - Route discovery service (RDS)\n            - Endpoint discovery service (EDS)\n            - Listener discovery service (LDS)",
-    "id": 1035172,
-    "modified": "2020-04-24T20:27:51.817384+00:00",
-    "new_id": "3y6-epj-nrz",
+    "id": 1170440,
+    "modified": "2020-06-09T16:55:14.604542+00:00",
+    "new_id": "6pb-hvt-xq7",
     "read_only": false,
-    "template_variables": [],
+    "template_variables": [
+        {
+            "default": "*",
+            "name": "cluster",
+            "prefix": "cluster_name"
```

Comment:
I think the vars are missing, you might need to click on `Add to All`, then export:

![image](https://user-images.githubusercontent.com/49917914/84178796-4f896c80-aa85-11ea-8968-4b4b4312efe9.png)